capstone/src/main/java/com/google/sps/Objects/Post.java

```diff
@@ -46,15 +46,19 @@ public String getImg() {
     return img;
   }
 
-  public ArrayList<User> getLikes() {
+  public ArrayList<String> getLikes() {
     return likes;
   }
 
   public void addComment(Comment newComment) {
     this.comments.add(newComment);
   }
 
-  public void addNewLike(User newLike) {
-    this.likes.add(newLike);
+  public void addLike(String userId) {
+    if (!likes.contains(userId)) this.likes.add(userId);
```

Comment:
This is good, but could you instead switch this over to using a Set instead of an ArrayList? Similar to the conversation we had during the daily stand-up on Thursday

Reply:
Ignore this comment for now. I'm looking into it based on Himani's comment in her PR about datastore only returning a list

Reply:
Just linking this incase you didn't see it: https://github.com/googleinterns/step100-2020/pull/8#discussion_r446358061
Instead of using an ArrayList change this to a HashSet, then you don't need to do the check if it contains, it will just not add it if it already does

Reply:
That makes sense to me, changed likes to a hashset.